master/reference/node/configuration.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Configuring calico/node
+---
+
+The `calico/node` container is primarily configured through environment variables.
+
+
+## Environment Variables
+
+| Environment | Description                              | Schema | Default |
+| ------------- | -------- | ------ | ----- |
+| NODENAME | A unique identifier for this host. | string | |
```

Comment:
When both the NODENAME and HOSTNAME environments are omitted it defaults to the hostname.